### DISCUSSION TITLE
Advantage of switching to ARM64 version?

### DISCUSSION THREAD

**Discussion by E-Hain** [thijsvanloef/palworld-server-docker#312] [Q&A] [answered]
What are the advantages or differences in switching from the default docker build to the ARM64 docker version? What is the default docker build running on? ARM32?

**Comment by thijsvanloef** [comment, marked as answer]
Hi @E-Hain, simply put, if you have the container running already, there is no advantage of switching to ARM64.
The ARM64 image is meant for ARM based architecture CPU's, CPU's you might find in raspberry pi's or M1 macs.
Most CPU's (Intel and AMD) are based on x64 architecture, and are recommended to use the regular `:latest` image, which is build on/for x64 CPU architecture

**Reply by E-Hain** [reply]
Thank you for the explanation!
